### PR TITLE
Sources: fix sentese cases

### PR DESCRIPTION
**Pull request by rvsia**
Part of https://projects.engineering.redhat.com/browse/TPINVTRY-832

### CODE REVIEW

File: packages/sources/src/addSourceWizard/hardcodedComponents/aws/access_key.js [hunk 1]
```diff
@@ -8,14 +8,14 @@ Create an access key in your AWS user account and enter the details below.
...
-    the Power User Identity and Access Management (IAM) policy for your AWS user account.&nbsp;
+    the Power user identity and access management (IAM) policy for your AWS user account.&nbsp;
         <Popover
             aria-label="Help text"
             position="bottom"
             bodyContent={
                 <React.Fragment>
                     <Text component={ TextVariants.p }>
-                        The Power User IAM policy allows the user full access to API functionality and AWS services for user administration.
+                        The Power user policy allows the user full access to API functionality and AWS services for user administration.
```
Comment (Hyperkid123) [inline packages/sources/src/addSourceWizard/hardcodedComponents/aws/access_key.js]: Should not this be `The power user`?

File: packages/sources/src/addSourceWizard/hardcodedComponents/aws/access_key.js [hunk 2]
```diff
@@ -8,14 +8,14 @@ Create an access key in your AWS user account and enter the details below.
...
 Create an access key in the Security Credentials area of your AWS user account.
```
Comment (Hyperkid123) [inline packages/sources/src/addSourceWizard/hardcodedComponents/aws/access_key.js]: Also `Security Credentials`. Don't know if the sentence case applies here

File: packages/sources/src/addSourceWizard/hardcodedComponents/aws/access_key.js (hunk 1 above)
Comment (rvsia) [reply packages/sources/src/addSourceWizard/hardcodedComponents/aws/access_key.js]: Not sure, I am following the JIRA description.

Comment (Hyperkid123) [reply packages/sources/src/addSourceWizard/hardcodedComponents/aws/access_key.js]: Ok.

File: packages/sources/src/addSourceWizard/hardcodedComponents/aws/access_key.js (hunk 2 above)
Comment (rvsia) [reply packages/sources/src/addSourceWizard/hardcodedComponents/aws/access_key.js]: Same answer as above ^^ :D

Comment (Hyperkid123) [reply packages/sources/src/addSourceWizard/hardcodedComponents/aws/access_key.js]: Well if you have description from someone. I think its good.

**Pull request merged**